zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/DocumentControllerTest.java

```diff
@@ -189,6 +194,129 @@ void shouldCreateDocumentWithMetadata() {
     assertThat(metadata.contentType()).isEqualTo(contentType.toString());
   }
 
+  @Test
+  void shouldCreateDocumentsBatchWithMetadataNumberKeys() throws JsonProcessingException {
```

Comment:
❓ Are we using number keys here? I cannot see number keys, I can only see string keys. But maybe I am missing something 😅 

💭 We could actually test with number keys as input, if we wanted to, I think. But we'd have to write the form data as a plain string then, I guess, to create the JSON part manually and not with the data model (that takes a string key).

Reply:
No, this was just meant as a simple test for batch creation. I'll use a name without `NumberKeys`.